docs/Demo/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from '@deque/cauldron-react/';
+
+const Footer = props => {
+  const year = new Date().getFullYear();
+
+  return (
+    <footer
+      className={
+        props.theme === 'dark' ? 'cauldron--theme-dark Footer' : 'Footer'
```

Comment:
Why is this needed? The body class should already have the `cauldron--theme-*` class

Reply:
Good catch!